tests/behavior/transforms/pipelines_complete.toml

```diff
@@ -21,7 +21,7 @@ order = [
 [transforms.processing.logs.pipelines.aws_s3_access_logs]
 name = "AWS S3 access logs"
 filter.type = "datadog_search"
-filter.condition = "source:s3"
+filter.source = "source:s3"
```

Comment:
Is this right? I'm not seeing where this field is mapped in the config.

Reply:
Considering I'm using the `AnyCondition` structure, if I follow [this example](https://github.com/vectordotdev/vector/blob/master/src/conditions/mod.rs#L120), we need to use the `source` attribute 🤔

Reply:
Oh, duh, yes. Sorry, I forgot that field was called `source`.